### PR TITLE
Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 702 #16173

### PR DESCRIPTION
**Pull request by Manimaran11**
#### Summary
Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 702 #16173

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/16173

### CODE REVIEW

File: sass/layout/_headers.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
     font-size: 14px;
     position: relative;
     width: 100%;
+    fill :rgba(var(--center-channel-color-rgb), 0.6);
```
Comment (asaadmahmood) [inline sass/layout/_headers.scss:12]: Please remove the extra spcae between `fill` and the `:` in both places and add a space after `:`

**Review by asaadmahmood — CHANGES_REQUESTED** [review]

File: sass/layout/_headers.scss (hunk 1 above)
Comment (Manimaran11) [reply sass/layout/_headers.scss]: please check now

**Review by asaadmahmood — APPROVED** [review]

File: sass/layout/_headers.scss [hunk 2]
```diff
@@ -9,6 +9,7 @@
     font-size: 14px;
     position: relative;
     width: 100%;
+    fill: rgba(var(--center-channel-color-rgb), 0.6);
```
Comment (calebroseland) [inline sass/layout/_headers.scss:12]: What purpose does this serve? The original `changeCss` doesn't include this one.

**Review by calebroseland — APPROVED** [review]
@Manimaran11 thanks! LGTM

**Pull request merged**